### ISSUE TITLE
Interemptor Squad

### ISSUE THREAD

**Issue by Doreo** [BSData/horus-heresy#2161]
Misslie Launchers and Plasma Incinerators should cost 15pts each (atm No Cost)

Melta Bombs and Artificier Armour upgrades for the Praefectus should cost 10pts each (atm No Cost)

Interemptor Praefectus should cost 25pts (now No Cost and there is an hidden 25pts cost built in instead).

**Comment by revivedtitan** [comment]
Good catch. I made the updates and will push them in my next pull request. We usually merge all the pull requests once a week so you wont see the updates until next week most likely

**Comment by nstephenh** [comment]
Reference this issue in the pull request please, just for tracking
purposes.
